pdbApp/pdbgroup.cpp

```diff
@@ -349,12 +350,24 @@ void PDBGroupPut::put(pvd::PVStructure::shared_pointer const & value,
     // assume value may be a different struct each time... lot of wasted prep work
     const size_t npvs = channel->pv->members.size();
     std::vector<std::tr1::shared_ptr<PVIF> > putpvif(npvs);
+    std::vector<void *> asWritePvt;
 
     for(size_t i=0; i<npvs; i++)
     {
         PDBGroupPV::Info& info = channel->pv->members[i];
-        if(!info.allowProc) continue;
```

Comment:
Is this removal intentional?

Reply:
Hi,
thank you for review.

This was not removed, only GitHub is marking it badly, it still there bellow added code: https://github.com/epics-base/pva2pva/pull/37/files#diff-bd4da3f325a325509ce78293ec376f10R370

I was not sure if trap should come before or after this call. This way notification is sent also if user is not allowed to write if I understand correctly.